include/core/parameters.h

```diff
@@ -1238,7 +1236,7 @@ namespace Parameters
       kelly
     } type;
 
-    /// Field influencing the mesh adaptation
+    /// Fields on which the mesh adaptation can be based
```

Comment:
```suggestion
    /// Fields on which the mesh adaptation can be based.
```

Reply:
Fun fact : the period is added automatically on the doc :)
![image](https://github.com/lethe-cfd/lethe/assets/107414376/7d1be068-effc-4b23-a0b0-cd16661e0c20)

Reply:
Good to know!